demo.py

```diff
@@ -93,8 +100,8 @@ def train_model(X_train, y_train, X_test, y_test, lmd):
     y_train_pred = clf.predict(X_train)
     y_test_pred = clf.predict(X_test)
 
-    print('accu (train): %5.2f'%(metric(y_train, y_train_pred)))
-    print('accu (test): %5.2f'%(metric(y_test, y_test_pred)))
+    print('accu (train): {:5.2f}'.format(metric(y_train, y_train_pred)))
+    print('accu (test): {:5.2f}'.format(metric(y_test, y_test_pred)))
 
     # Ensembles of Decision Tree
     print('\nDecision tree')
```

Comment:
Please take a look at the latest output (https://github.com/dwave-examples/qboost/pull/12#issuecomment-767660537) and see if you have other suggestions.  I have updated it so that it lists the method name followed by the output for training and test accuracy.